.github/workflows/ci.yml

```diff
@@ -1,8 +1,5 @@
 name: Tests
 on:
-  push:
-    branches:
-      - main
   pull_request:
```

Comment:
```suggestion
  pull_request:
  workflow_call:
```

Reply:
Can you please also rename the file to ci.yaml to match release.yaml and other DPE repos. Tnx!